include/zephyr/tracing/tracing.h

```diff
@@ -1870,17 +1870,17 @@
  * @brief Trace posting of an Event call entry
  * @param event Event object
  * @param events Set of posted events
- * @param accumulate True if events accumulate, false otherwise
+ * @param events_mask Mask to apply against posted events
  */
-#define sys_port_trace_k_event_post_enter(event, events, accumulate)
+#define sys_port_trace_k_event_post_enter(event, events, events_mask)
```

Comment:
This basis of this comment also applies to the sys_port_trace_k_event_post_exit() change.

Almost there. There are a couple of other references to sys_port_trace_k_event_post_enter() has some references in ...

subsys/tracing/test/tracing_test.h
subsys/tracing/user/tracing_user.h
subsys/tracing/ctf/tracing_ctf.h

That **accumulate** parameter in those files will also need to be updated to **events_mask** as well.

Reply:
Sorry I missed those: thank you for your patience (again)